app/lib/rocketchat.js

```diff
@@ -1000,6 +1000,31 @@ const RocketChat = {
 			searchText
 		});
 	},
+	toggleOwner(roomId, userId, owner, t) {
+		// RC 1.0
```

Comment:
Where did you get this server version from?
Take a look at our docs: https://rocket.chat/docs/developer-guides/rest-api/channels/addowner/#change-log
It was introduced on version 0.49.4.

Reply:
@diegolmello I have gone through the docs to get the api endpoints but I forgot to change server version. Sorry for that